test_util/integration_test.py

```diff
@@ -250,15 +261,15 @@ def __init__(self, dcos_uri, masters, public_masters, slaves, public_slaves, reg
         # Make URI never end with /
         self.dcos_uri = dcos_uri.rstrip('/')
 
-        self.is_oauth = os.environ.get('DCOS_OAUTH', 'true') == 'true'
+        self.auth_enabled = auth_enabled
```

Comment:
This is always set to AUTH_ENABLED the global, so what value are we getting setting it to a member anymore?